src/shared/components/timer/timer.component.js

```diff
@@ -0,0 +1,123 @@
+import React, { PureComponent } from 'react';
+import moment from 'moment';
+import firebase from 'firebase';
+
+import { StartButton, SaveButton, Time, HistoryContainer } from './timer.styles';
+
+require('firebase/firestore');
```

Comment:
```suggestion
import 'firebase/firestore';
```
W przypadku, gdybyś miał service to ten import wylądowałby właśnie tam.